public/app/types/jest/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Observable } from 'rxjs';
```

Comment:
nit: would it be possible to use the same definition file as we use in grafana/data?

Reply:
Oh noes, I forgot to remove that one. Great catch!!